jnigen/pubspec.yaml

```diff
@@ -2,13 +2,16 @@
 # for details. All rights reserved. Use of this source code is governed by a
 # BSD-style license that can be found in the LICENSE file.
 
+# DO NOT MERGE: prototype.
+publish_to: none
+
 name: jnigen
 version: 0.3.0
 description: Experimental generator for FFI+JNI bindings.
 repository: https://github.com/dart-lang/jnigen/tree/main/jnigen
 
 environment:
-  sdk: '>=2.17.0 <4.0.0'
+  sdk: ">=2.17.0 <4.0.0"
```

Comment:
Ah we saw this before.

I set YAML formatting >> single quotes in VS code. Should not matter though.